### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by mwanawabangona**
My-Portfolio, Popup window

 -Add Html file
 -Add js file
 -Add CSS file
 -Add Github folder in which includes linters.yml file
 -Add gitignore file in which includes node-module/ and Vscode files
 -Add Linters config files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,375 +1,341 @@
...
+                            </div>
+                            <p class="card-text">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+
+                            <div class="tags">
+                                <span class="badge rounded-pill bg-tag">html</span>
+                                <span class="badge rounded-pill bg-tag">css</span>
+                                <span class="badge rounded-pill bg-tag">javascript</span>
+                            </div>
```
Comment (Davidosky007) [inline index.html]: * The technologies tag are overflowing hence causing the popup window to stretch for both desktop and mobile views kindly fix it 🙏 
image
image

File: js/popup.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  detail1: {
+    name: 'Tonic',
+    img: 'https://codecaiine.github.io/microverse-javascript/image/portfolio1.png',
+    companyName: 'CANOPY',
+    job: 'Back End Dev',
+    year: 2015,
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    tags: ['html', 'css', 'javaScript'],
```
Comment (Davidosky007) [inline js/popup.js:10]: * Kindly add the link to the live version and link to source in each object as it is stated in the project requirement it should be included here in each object [Refer to the project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=For%20each%20project%20you%20need%20to%20store%20the%20following%20pieces%20of%20data%20in%20a%20JavaScript%20object%2C%20at%20a%20minimum%3A%20name%2C%20description%2C%20featured%20image%2C%20technologies%2C%20link%20to%20live%20version%2C%20link%20to%20source.)

File: index.html [hunk 3]
```diff
@@ -1,375 +1,341 @@
...
+                            <div class="mb-1 text-muted work-info">
+                                <ul>
+                                    <li class="intro">FACEBOOK</li>
+                                    <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p class="card-text">
+                                Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                            </p>
```
Comment (Davidosky007) [inline index.html]: * Kindly load each project content from the array of objects created and not hard code it using `HTML` doing so would make addition and subtraction of projects in the future easy plus it is a requirement for this project [Check Here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=For%20each%20project,the%20page%20loads.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @mwanawabangona 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly write your commit message in the imperative mood (ie Fix and not Fixed or Fixes) this is the best practice [Read more here on how to properly write commit messages ](https://medium.com/@steveamaza/how-to-write-a-proper-git-commit-message-e028865e5791)
* I suggest you use the project object created to populate the project section on page load and not hard code it with `HTML` check comments below and make necessary changes 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-N.A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (bellom) [reply index.html]: @mwanawabangona,

Mobile Version:
Let's give some margin between span and the buttons.
Let's fix the border for the button too, kindly make it look like the original design,

Original Design:
image

Your app:
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @mwanawabangona  👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there is still some little styling to fix.

 ## Required Changes ♻️
 
_Kindly check my comment above this review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemantso — APPROVED** [review]
Hi @mwanawabangona, 

Good work on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**